### PR TITLE
Add basic setup functionality to MCD

### PR DESCRIPTION
**Pull request by hexonaut**
Should probably add tests, but I'm using it in dss-bridge.

### CODE REVIEW

**Comment by gbalabasquer** [comment]
This overlaps a bit with what `DssDeploy` does.
It's true that the main value of `DssDeploy` is for an actual deployment, however we've been using the [test base](https://github.com/makerdao/dss-deploy/blob/master/src/DssDeploy.t.base.sol#L227) for testing in other repos a lot as well.

This is though more light weighted and actually allows you to deploy whatever you want as `vat`, `vow` and the other core modules, so I see it has a different angle.

**Comment by gbalabasquer** [comment]
I think we should avoid using `DSValue` dependency and just copy and paste the functionality here (reduced to the maximum).
Otherwise we are attaching ourselves back to a dapphub dependency.
We are still doing through `dss-interfaces` but I guess that will change at some point.

**Comment by hexonaut** [comment]
Done.

**Comment by gbalabasquer** [comment]
Thanks reviewing, and actually `dss-interfaces` doesn't rely on dapphub repos at all, so we should be perfectly fine now.

File: src/MCD.sol [hunk 1]
```diff
@@ -18,6 +18,30 @@ pragma solidity >=0.8.0;
 import "dss-interfaces/Interfaces.sol";
 
 import {MCDUser} from "./MCDUser.sol";
+import {GodMode} from "./GodMode.sol";
```
Comment (gbalabasquer) [inline src/MCD.sol:21]: Add above Dai Foundation copyright this one:
```
// Copyright (C) 2017  DappHub, LLC
```

**Review by gbalabasquer — APPROVED** [review]
LGTM

**Pull request merged**